{{cookiecutter.project_shortname}}/.travis.yml

```diff
@@ -42,41 +35,21 @@ env:
     - REQUIREMENTS=qa
 
 python:
-  - "3.6"
+  - "3.5"
```

Comment:
https://m.imgur.com/t/reaction/vrsLBmp

Joke aside, Python 3.5 has a similar EOL to 2.7 so maybe it's worth pushing for 3.6...

Reply:
I know, I use this one because it's 3.5 everywher! cookiecutter `.travis`, `Dokerfile`, `setup.py` ...